include/psa/crypto_se_driver.h

```diff
@@ -924,41 +928,51 @@ typedef psa_status_t (*psa_drv_se_export_key_t)(psa_drv_se_context_t *drv_contex
  * element
  *
  * If \p type is asymmetric (#PSA_KEY_TYPE_IS_ASYMMETRIC(\p type) = 1),
- * the public component of the generated key will be placed in `p_pubkey_out`.
- * The format of the public key information will match the format specified for
- * the psa_export_key() function for the key type.
+ * the driver may export the public key at the time of generation,
+ * in the format documented for psa_export_public_key() by writing it
+ * to the \p pubkey buffer.
+ * This is optional, intended for secure elements that output the
+ * public key at generation time and that cannot export the public key
+ * later. Drivers that do not  need this feature should leave
+ * \p *pubkey_length set to 0 and should
+ * implement the psa_drv_key_management_t::p_export_public function.
+ * Some implementations do not support this feature, in which case
```

Comment:
NULL can also be passed in a case when the user is not interested in receiving the public key, right?

Reply:
No, there's no such thing as “the user is not interested in receiving the public key”. `psa_generate_key` never outputs the public key. If the driver outputs the public key, the core stores it in case the user ever calls `psa_export_public_key` later.